### DISCUSSION TITLE
Achieving Diversity in Automated Character Generation: Beyond Manual Seed Adjustments

### DISCUSSION THREAD

**Discussion by VintageGenetics** [lllyasviel/Fooocus#2203] [Q&A] [answered]
I want to automatically generate a unique individual for each instance, up to 20 variations. Currently, hitting generate produces 20 variations of the same person, but I aim for diversity in the results, such as a Black woman, an Indian woman, and a White woman among the 20 outcomes. I understand that manually changing the seed for each generation is an option, but I'm looking for a more streamlined and efficient approach to achieve this.

**Comment by eddyizm** [comment, marked as answer]
Have you tried to use the `ethnicity` wildcard?

**Reply by goneli03** [reply]
Hey, I just wanted to ask and see if you could help me out. I recently got fooocus and I'm a big fan of it. I've seen many videos on it. The thing is that I'm on MacBook Air. Here's the info on my Mac: Chip Apple M2
Memory
8 GB
Startup disk
Macintosh HD
macOS
Sonoma 14.0
Fooocus works good but it is very slow, like over an hour to generate one image. What would you recommend me to do to possibly speed the process up? Or would it strictly be using windows ? Anything helps, thanks!

**Reply by mashb1t** [reply]
The Macbook you're using is sadly underspeced for Fooocus, sorry to say.
You can use Colab as a cloud tool free of charge. Please find the link here: https://colab.research.google.com/github/lllyasviel/Fooocus/blob/main/fooocus_colab.ipynb

**Reply by VintageGenetics** [reply]
Thank you, I haven't and I shall! :)
